lighty-core/lighty-parent/pom.xml

```diff
@@ -181,7 +181,7 @@
                 <plugin>
                     <groupId>com.github.spotbugs</groupId>
                     <artifactId>spotbugs-maven-plugin</artifactId>
-                    <version>4.5.3.0</version>
+                    <version>4.7.2.0</version>
```

Comment:
Please update below section according to: https://mvnrepository.com/artifact/com.github.spotbugs/spotbugs-maven-plugin/4.7.2.0